Cargo.toml

```diff
@@ -11,13 +11,14 @@ git = "https://github.com/dwrensha/capnpc-rust"
 log = "*"
 rand = "*"
 uuid = "*"
+bufstream = "*"
 
 [dependencies.capnp]
 git = "https://github.com/danburkert/capnproto-rust"
 branch = "async"
 
 [dependencies.mio]
-git = "https://github.com/carllerche/mio"
+git = "https://github.com/danburkert/mio"
```

Comment:
This was at least for the usage of `Handle::ready` instead of the `Handle::readable,writable`. Doesn't appear to have any PR to merge. If it is to get merged, I would recommend that for compatibility reasons that `ready` implements the old functionality with all callbacks. It should also be noted that `ReadHint` no longer exists.